api/desecapi/crypto.py

```diff
@@ -1,10 +1,24 @@
 from base64 import urlsafe_b64encode
 from hashlib import sha256
 
+from cryptography.fernet import Fernet, InvalidToken
 from django.conf import settings
 from django.core.signing import Signer
 
 SECRET_KEY_SIGN = urlsafe_b64encode(sha256(('sign-' + settings.SECRET_KEY).encode()).digest())
+SECRET_KEY_CRYPT = urlsafe_b64encode(sha256(('encrypt-' + settings.SECRET_KEY).encode()).digest())
+fernet = Fernet(key=SECRET_KEY_CRYPT)
+
+
+def encrypt(data):
+    return fernet.encrypt(data)
+
+
+def decrypt(token):
+    try:
+        return fernet.decrypt(token)
+    except InvalidToken:
+        raise ValueError
```

Comment:
remove try/except: it doesn't do anything but changing the exception type

Reply:
The intention was that the code that uses this and wants to catch the exception (serializers.py) does not need to import `from cryptography.fernet import InvalidToken` in order to return a sensible error. Does that make sense?